webapp/src/index.tsx

```diff
@@ -53,12 +54,15 @@ export default class Plugin {
 
         registry.registerWebSocketEventHandler(`custom_${manifest.id}_connect`, handleConnect(store));
         registry.registerWebSocketEventHandler(`custom_${manifest.id}_disconnect`, handleDisconnect(store));
+        registry.registerWebSocketEventHandler(`custom_${manifest.id}_link_channels`, handleModalLink(store));
+        registry.registerWebSocketEventHandler(`custom_${manifest.id}_link`, handleLink(store));
     }
 }
 
 declare global {
     interface Window {
-        registerPlugin(id: string, plugin: Plugin): void,
+        registerPlugin(id: string, plugin: Plugin): void;
+        Components: any;
```

Comment:
check if we have type for this

Reply:
https://github.com/mattermost/mattermost/blob/master/webapp/channels/src/plugins/export.js

Components is having type `any` in above file